scripts/utils/events.js

```diff
@@ -56,11 +56,19 @@ const projectsEvents = () => {
   projectForm.addEventListener("submit", projectsFormSubmit);
 };
 
-
-
-
-const overviewEvents = () => {
-  return;
+const overviewEvents = (e) => {
+  e.preventDefault()
+  if(e.target.id === 'pinRepoForm'){
+    const pinIndex = document.querySelector('#pinRepoSelect').value;
+    repositories[pinIndex].isPinned = true;
+    overviewCardBuilder(repositories);
+    overviewFormBuilder(repositories);
+  } else if (e.target.id.includes('unpinRepo')){
+    const unpinIndex = e.target.id.split("---")[1];
```

Comment:
If a repo is unpinned, I update the repo array and because the form and the cards depend on that data, I rerender those so that can be updated.